easier68k/core/opcodes/adda.py

```diff
@@ -0,0 +1,309 @@
+from ...core.enum.ea_mode import EAMode
+from ...core.enum import ea_mode_bin
+from ...core.opcodes.opcode import Opcode
+from ...simulator.m68k import M68K
+from ...core.enum.ea_mode_bin import parse_ea_from_binary
+from ...core.models.assembly_parameter import AssemblyParameter
+from ...core.util import opcode_util
+from ..util.parsing import parse_assembly_parameter
+from ...core.enum.op_size import OpSize
+from ..util.split_bits import split_bits
+from ..models.memory_value import MemoryValue
+
+class Adda(Opcode):
+    pass
+
+class Adda(Opcode):
+
+    # Allowed sizes for this opcode
+    valid_sizes = [OpSize.WORD, OpSize.LONG]
+
+    def __init__(self, params: list, size: OpSize = OpSize.WORD):
+        assert len(params) == 2
+        assert isinstance(params[0], AssemblyParameter)
+        assert isinstance(params[1], AssemblyParameter)
+        # Can't take data register, address register, or ARI with modifications
+        # assert params[0].mode not in [EAMode.DRD, EAMode.ARD, EAMode.ARIPD, EAMode.ARIPI, EAMode.IMM]
```

Comment:
Unintentional commented out code?

Reply:
The source destination is an effective address that allows for all modes so this can just be deleted.